src/doc/trpl/concurrency.md

```diff
@@ -343,12 +343,12 @@ threads as a simple isolation mechanism:
 ```rust
 use std::thread;
 
-let result = thread::spawn(move || {
+let handle = thread::spawn(move || {
     panic!("oops!");
-}).join();
+});
 
-assert!(result.is_err());
+assert!(handle.join().is_err());
```

Comment:
(like, add a line above with `let result = handle.join()` and then `assert!` on `result.is_err()`.